src/views/AIView.tsx

```diff
@@ -1,7 +1,10 @@
 import { Command } from '@tauri-apps/api/shell';
 import { useEffect, useState } from 'react';
 
-export function AIView({ selection }: { selection: string | null }) {
+import { useAppSelector } from '../redux/hooks';
+
+export function AIView({ selection: x }: { selection: string | null }) {
```

Comment:
You should remove `selection` from the props, we aren't using it anymore.

Reply:
That was a typo. Good catch.